media/css/mozorg/home/home-promo.less

```diff
@@ -866,10 +866,10 @@ html[lang|="en"] {
     background: #0ad; // bright blue
 
     h2 {
-        padding: 30px 20px 10px 140px;
+        padding: @baseLine 20px 10px 140px;
```

Comment:
These changes are breaking an older web maker promo that's already on the page:

![teach-the-web](https://cloud.githubusercontent.com/assets/400117/11042338/66aead2a-870e-11e5-9574-148fa9a70f2b.png)

I think we should probably use a new class name for this newer promo, instead of altering existing ones.